### ISSUE TITLE
[ID] Suppress the "What happens after probation/parole?" sections 

### ISSUE THREAD

**Issue by hobuobi** [Recidiviz/public-dashboard#595]
For the "What happens after probation/parole" sections, Spotlight currently uses a metric that is meant to communicate the % of people who successfully complete supervision in a given month. However, this uses on a complicated methodology that we frequently do not have the data for (and definitely do not in Idaho, resulting in the chart breaking).

For now, we should suppress these sections. This will unblock the launch and allow us to update the chart in the future with a new metric or methodology.

In the future, we probably want to replace this with a chart showing normalized revocation rates from supervision (parole/probation) per month. This will require some work from MINDMELD (@samanthahuff) to produce the relevant data; the urgency of this work is low, unless there is a request from IDOC or we have extra bandwidth at some point down the line.

**Comment by terryttsai** [comment]
Is there a follow-up issue for what we want to do in the future?
